recipes/cfm-id/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set CONDA_BOOST = "1.66" %}
+package:
+  name: cfm-id
+  version: "33"
+
+source:
+  # the download is a mirror of a snapshot of 
+  # https://sourceforge.net/code-snapshots/svn/c/cf/cfm-id/code/cfm-id-code-r33-cfm.zip
+  # which is only present for 24h or so
+  url: https://depot.galaxyproject.org/software/cfm/cfm_r33_src_all.zip
+  md5: f02c79016eae6ad95413213509513eed
+  patches:
+    - CMakeLists.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - boost
```

Comment:
boost seemed plausible to me since this is also what the conda forge rdkit requirement uses (https://github.com/conda-forge/rdkit-feedstock/blob/master/recipe/meta.yaml). What's the difference?

Btw. the bioconda rdkit recipes are blacklisted and could be removed since conda forge has recipes. Question is if the eden and chemfp bioconda recipes survive this...?

Reply:
boost is boost-cpp plus python I think.
rdkit should really be removed. All others sould survive and chemfp could probably also be migrated if needed/interest exists.

Reply:
you are correct. So I change to boost-cpp, because its whats actually needed by cfm. However it won't make a difference since rdkit requires boost.